parametric/gen.py

```diff
@@ -838,7 +843,7 @@ def make_clones(self, turtle, seg_ind, split_corr_angle, num_branches_factor, cl
                 turtle.right.normalize()
 
             # create new clone branch and set up then recurse
-            split_stem = self.branches_curve.splines.new('BEZIER')
+            split_stem = self.branch_curves[stem.depth].splines.new('BEZIER')  # TODO :: Set depth here
```

Comment:
What's the TODO here for? looks like we are adding at the correct level?

Reply:
Crap. Because I neglected to ctrl+f for TODOs before making the PR.